gallery/gallery/lib/data/demos.dart

```diff
@@ -38,6 +38,8 @@ import 'package:gallery/demos/reference/typography_demo.dart';
 import 'package:gallery/l10n/gallery_localizations.dart';
 import 'package:gallery/themes/material_demo_theme_data.dart';
 
+const docsBaseUrl = 'https://api.flutter.dev/flutter';
```

Comment:
nit: this should probably be private to this class.

```suggestion
const _docsBaseUrl = 'https://api.flutter.dev/flutter';
```

Reply:
oops, fixed